fix: 565-whats-new-images

andreancardona opened the pull request:
Closes #565

vpicone commented:
Nice, good pick up. You picked the right spot for the style, but could we use css modules? Just make a “Grid.module.scss” under styles and import it like you would for a component. Bonus points if we do that for the other “grid_temp” classes and nuke this nasty style sheet.

sstrubberg commented:
I'm not seeing any changes being made to the images on the What's New page. (using Chrome)

![image](https://user-images.githubusercontent.com/15822070/84786175-015af880-afb2-11ea-8f40-bfe8ee834198.png)

andreancardona commented:
> I'm not seeing any changes being made to the images on the What's New page. (using Chrome)
> 
> ![image](https://user-images.githubusercontent.com/15822070/84786175-015af880-afb2-11ea-8f40-bfe8ee834198.png)

@sstrubberg oh yeah I converted it to draft and updated the module. I just pushed up all the changes - hopefully you see it now :)

mjabbink commented:
@andreancardona Looks good.

vpicone commented:
@andreancardona could you figure out what's up with this dang philosophy stylesheet?

mjabbink commented:
Still seeing the issue in mobile

mjabbink commented:
oh sorry. This hasn't merged yet. Looks good in preview.

andreancardona commented:
> @andreancardona could you figure out what's up with this dang philosophy stylesheet?

fixed! @vpicone

vpicone commented on src/styles/Grid.module.scss:
```diff
@@ -0,0 +1,123 @@
+  background: $carbon--white-0;
+}
+
+.bx--aspect-ratio--image .gatsby-resp-image-wrapper {
```
if possible we want to avoid styling this, If an image is added without gatsby-image (e.g. imported and used as `src`) it won't have this class.

vpicone replied:
Let me know if you're not able to get that workin!

sstrubberg commented:
FYI: I noticed this same issue on w3/Design regarding images with links rendered via markdown. I'm curious if there is something that needs to be done to the theme itself rather than add a custom class. @vpicone

sstrubberg commented on src/styles/Grid.module.scss:
```diff
@@ -0,0 +1,123 @@
+  left: 0 !important;;
+}
+
+.bx--aspect-ratio--image {
```
.#{$prefix} instead of the .bx

vpicone commented:
@sstrubberg yeah I'm not sure, we could gather up some examples of folks doing link-images and figure out a theme solution. It seems like the markdown strat @sadekbazaraa figured out has been working okay.

@andreancardona For some reason I thought this was the theme repo, while targeting the `gatsby-resp-image-wrapper` is a little hacky for the theme (where we can't control how folks are providing images), I think it's fine here since we know they're all going to be provided via markdown (which will use gatsby-image and give us this class)

sstrubberg commented on src/styles/Grid.module.scss:
```diff
@@ -0,0 +1,123 @@
+    }
+  }
+
+// grid classnames (should be added to either grid package or theme?)
```
I thought all of these no gutter styles were already in the theme?

andreancardona replied:
okay it actually seams these styles weren't doing anything...so just deleted them for now.

andreancardona commented:
@sstrubberg @vpicone ready for you both to review!

vpicone commented:
🚀

mjabbink commented:
👍
